### ISSUE TITLE
[BottomNavigation] [iOS] Having selected index in XML does not change correctly styles

### ISSUE THREAD

**Issue by PeterStaev** [nativescript-community/ui-material-components#271]
### If the demo apps cannot help and there is no issue for your problem, tell us about it
Having a default selected index defined in the XML does not change correctly the styles of the tabs, although the tab is loaded correctly. Happens only on iOS, on android works perfectly fine. 

### Which platform(s) does your issue occur on?
- iOS
- 14.4
- emulator

### Please, provide the following version numbers that your issue occurs with:

- CLI: 8.0.1
- Cross-platform modules: 8.0
- Runtime(s): 8.0
- Plugin(s): (look for the version numbers in the `package.json` file of your
project and paste your dependencies and devDependencies here)

### Please, tell us how to recreate the issue in as much detail as possible. 
Describe the steps to reproduce it.

### Is there any code involved? 
Sample XML:
```xml
<Page xmlns="http://schemas.nativescript.org/tns.xsd" 
xmlns:mdt="@nativescript-community/ui-material-bottom-navigation"
navigatingTo="navigatingTo">

    <ActionBar title="My App" icon="" />

    <GridLayout>
        <mdt:BottomNavigation  
            id="view-bottom-nav"
            style="font-size: 15;" 
            selectedIndex="1">
            
            <!-- The bottom tab UI is created via TabStrip (the containier) and TabStripItem (for each tab)-->
            <mdt:TabStrip iosIconRenderingMode="alwaysOriginal">
                <mdt:TabStripItem class="tab" title="Client"  style="color:#fefefe;"/>
                <mdt:TabStripItem class="tab" title="Tasks" style="color:#fefefe;"/> 
            </mdt:TabStrip>
            <mdt:TabContentItem>
                <Label text="Tab 1" />
            </mdt:TabContentItem>
            <mdt:TabContentItem>
                <Label text="Tab 2" />
            </mdt:TabContentItem>

        </mdt:BottomNavigation>
    </GridLayout>
</Page>
```
CSS:
```css
.tab {
    background-color: blue;
    color: white;
}

.tab:active {
    background-color: red;
    color: white;
} 

```

**Comment by PeterStaev** [comment]
Something that also might be related: I'm selecting a tab via tap, the style gets applied correctly. Then I suspend the app via the home button, and reopen the app - the selected style gets lost.

**Comment by farfromrefug** [comment]
@PeterStaev that was working with N Tabs in 7.x?

**Comment by PeterStaev** [comment]
@farfromrefug, yes it was working fine with the built in tabs pre-NS-8.

**Comment by farfromrefug** [comment]
@PeterStaev fixed in 5.2.19

**Comment by PeterStaev** [comment]
@farfromrefug thanks for the fast fix, works as expected now!
